src/meta/proto-conv/src/util.rs

```diff
@@ -63,6 +63,7 @@ const META_CHANGE_LOG: &[(u64, &str)] = &[
     ),
     (15, "2022-10-12: Remove: precision in TimestampType"),
     (16, "2022-09-29: Add: CopyOptions::split_size"),
+    (17, "2022-10-28: Add: FileFormatOptions::escape"),
```

Comment:
This will conflict with another PR:
- #8526

Reply:
changed to 18.